shared-bindings/wifi/Radio.c

```diff
@@ -153,6 +170,40 @@ const mp_obj_property_t wifi_radio_hostname_obj = {
               (mp_obj_t)&mp_const_none_obj},
 };
 
+//|     def start_ap(self,
+//|                  ssid: ReadableBuffer,
+//|                  password: ReadableBuffer = b"") -> None:
+//|         """Starts an Access Point with the specified ssid and password."""
```

Comment:
I'd note here whether station connections are interrupted.

Reply:
I'll have to test this more. I thought at first glance that starting one might disrupt the other already running. If STA is running and AP is started (or vice versa), it becomes AP+STA and may not disrupt the original. They use separate `netif`s. I didn't see STA stops in the log when AP started.

Reply:
So far, it looks like the general case is that there is no disruption. I've only seen it once. If it happens more, we can characterize it as bug, or intended behavior under specific circumstances (to be documented).